website/docs/components/table/partials/code/how-to-use.md

```diff
@@ -545,6 +545,153 @@ In this case the table layout is still set to `auto` (default). If instead you w
 </div>
 ```
 
+### Multi-select table
```

Comment:
@hashicorp/hds-design @KristinLBradley @andgen404 should this be `Multi-select table` or `Multi-selectable table`? We should decide which one to use.

Reply:
+1 to `Multi-select table`

Reply:
"multi-select"